watch_connectivity_garmin/android/src/main/kotlin/dev/rexios/watch_connectivity_garmin/WatchConnectivityGarminPlugin.kt

```diff
@@ -64,6 +63,10 @@ class WatchConnectivityGarminPlugin : FlutterPlugin, MethodCallHandler {
         val applicationId = call.argument<String>("applicationId")!!
         iqApp = IQApp(applicationId)
 
+        val isTethered = call.argument<Boolean>("tethered")!!
+        var connectType = ConnectIQ.IQConnectType.WIRELESS
```

Comment:
We should actually represent this enum since there could be more values in the future

Reply:
implemented enum